src/main/java/frc/robot/commands/RunShooterToRPMCommand.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.commands;
+
+import frc.robot.subsystems.ShooterPrototypeSubsystem;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+/** An example command that uses an example subsystem. */
+public class RunShooterToRPMCommand extends CommandBase {
+  
+  private final ShooterPrototypeSubsystem shooterSubsystem;
+
+  /**
+   * Creates a new RunShooterToRPMCommand.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public RunShooterToRPMCommand(ShooterPrototypeSubsystem subsystem, double power) {
```

Comment:
We probably have to split into two commands, one that sets run/don't run, and one that changes the power by a certain amount. So we can have start and stop buttons do EnableShooterCommand(...,true) and EnableShooterCommand(...,false), and the X/Y buttons do ChangeShooterPowerCommand(..., 0.1) and ChangeShooterPowerCommand(..., -0.1).